inst/test_objects.R

```diff
@@ -1,4 +1,9 @@
+# ------------------------------------------------------------------------------
+# Run this, in place, from the terminal with R CMD BATCH --vanilla test_objects.R
+# Make sure to commit the Rout file
```

Comment:
This currently clashes with the bullet point for the release questions which suggests running R CMD BATCH in the pkg root directory (which is why it had all the paths constructed with `testthat::test_path()`. I don't have strong opinions on which way we do it, but it should be consistent :D 

https://github.com/tidymodels/tune/blob/a86603c0f3ad39b59d88c11977560a20cc78a759/R/0_imports.R#L57